backend/src/graphql/resolvers/UserResolver.spec.ts

```diff
@@ -100,6 +100,227 @@ describe('UserResolver', () => {
+                      username: 'tom',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Thomas Schmidt',
+                      username: 'Toms',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Tomas Schmid',
+                      username: 'Schmid',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Tomás Schmit',
+                      username: 'Schmit',
+                    },
+                  ],
+                },
+                errors: undefined,
+              },
+            },
+          })
+        })
+
+        it('returns the correct users for "TOM"', async () => {
+          const response = await testServer.executeOperation(
+            {
+              query: `query ($searchString: String) {users(searchString: $searchString) {id name username}}`,
+              variables: { searchString: 'TOM' },
+            },
+            {
+              contextValue: {
+                token: 'token',
+                dataSources: { prisma },
+              },
+            },
+          )
+
+          expect(response).toMatchObject({
+            body: {
+              kind: 'single',
+              singleResult: {
+                data: {
+                  users: [
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Thomas Schmitt',
+                      username: 'tom',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Thomas Schmidt',
+                      username: 'Toms',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Tomas Schmid',
+                      username: 'Schmid',
+                    },
+                    {
+                      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                      id: expect.any(Number),
+                      name: 'Tomás Schmit',
+                      username: 'Schmit',
+                    },
+                  ],
+                },
+                errors: undefined,
+              },
+            },
+          })
+        })
+
+        it('returns the correct users for "sch"', async () => {
```

Comment:
I don't see the point of this test case - what are you testing? Substring was considered above.

Reply:
I do not agree. We tested for strings starting with the search string. This test proves, that the search-string also matches in the middle of the string

Reply:
OK! "tom" is indeed the prefix. Then:
```suggestion
        it('filters by substring' async () => {
```